_posts/2021-08-02-debezium-1-7-alpha1-released.adoc

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+title:  Debezium 1.7.0.Alpha1 Released
+date:   2021-08-02
+tags: [ releases, mysql, postgres, sqlserver, cassandra, oracle, db2, vitess, outbox ]
+author: jpechane
+---
+
+It's my pleasure to announce the first release of the Debezium 1.7 series, *1.7.0.Alpha1*!
+
+With the summer in a full-swing, this release brings additional improvements to the Debezium Oracle connector but also to the others as well.
+
++++<!-- more -->+++
+
+== Alternative Oracle Connector LogMiner processors
+
+The Oracle connector can be memory-heavy in case of really large traffic with parallel large transactions in progress.
+All these changes in progress need to be kept in memory which might be impossible so in previous versions it was necessary to remove the really old transaction out of the picture.
+
+The current release brings significant refactoring of the connector code that enables the processor to be pluggable.
+There is the old memory-based available out-of-the-box but for the first time, we release a new processor based on https://infinispan.org/[Infinispan distributed data store].
+With this processor in place, it is possible to track an almost unlimited number of in-progress transactions for unlimited time.
+
+The first release has few limitations
+
+* the Inifinspan configuration is not exposed outside to the user for tuning
+* there is only one cache store provided - file-based which means that the connector needs either to run on a single machine or in case of a cluster the cache store must be located on a networked filesystem
```

Comment:
Doesn't the connector always run on a single machine? And what "cluster" are you referring to here?

Reply:
From my discussion with @jpechane, he was referring to a distributed KC cluster where the connector is run from one node to another.  I'm not an expert on KC and AK cluster topology, but he did mention that we want to consider providing users a way to fully configure the Infinispan setup via connector options or an external Infinispan config file reference for advanced users.

Reply:
Ah, thanks, I got it now. So the problem is rebalancing of connector tasks in a Kafka Connect cluster. Any node-local state (such as file-based Infinispan storage) will then be inaccessible. I think something like this may make it clearer:

> * Only file-based cache storage is supported; if you work with multi-node Kafka Connect clusters, a networked filesystem must be used for the cache store in order to support rebalancing of connector tasks in the Connect cluster

Reply:
This still needs adjustment?